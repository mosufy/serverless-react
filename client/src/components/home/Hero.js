import React, { Component } from 'react';

class Hero extends Component {
  render() {
    return (
      <header className="header-image">
        <div className="headline">
          <div className="container">
            <h1>Sher Tuition</h1>
            <h2>Improving Life Together</h2>
          </div>
        </div>
      </header>
    );
  }
}

export default Hero;