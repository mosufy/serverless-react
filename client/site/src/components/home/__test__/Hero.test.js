import React from 'react';
import renderer from 'react-test-renderer';

import Hero from '../Hero';

describe('home.Hero component testing', () => {
  it('it renders correctly', () => {
    const tree = renderer.create(
      <Hero />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});