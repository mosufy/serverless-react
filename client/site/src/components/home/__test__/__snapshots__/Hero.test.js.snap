// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home.Hero component testing it renders correctly 1`] = `
<header
  className="header-image"
>
  <div
    className="headline"
  >
    <div
      className="container"
    >
      <h1>
        Sher Tuition
      </h1>
      <h2>
        Improving Life Together
      </h2>
    </div>
  </div>
</header>
`;
